include/z64save.h

```diff
@@ -4,6 +4,32 @@
 #include "ultra64.h"
 #include "z64math.h"
 
+typedef enum {
+    /* 0x0 */ MAGIC_STATE_IDLE, // Regular gameplay
+    /* 0x1 */ MAGIC_STATE_CONSUME_SETUP, // Sets the speed in which magic border flashes
+    /* 0x2 */ MAGIC_STATE_CONSUME, // Consume magic until target is reached or no more magic is available
+    /* 0x3 */ MAGIC_STATE_BORDER_CHANGE_1, // Flashes border and freezes Dark Link
```

Comment:
I think for the name we can abstract that it's just the border flashing and not the whole meter, to name this like `meter_flash`?